projects/plugins/crm/tests/php/bootstrap.php

```diff
@@ -6,19 +6,61 @@
  */
 
 /**
- * Include the composer autoloader.
+ * Assume we're in tests/php/bootstrap.php.
  */
-require_once __DIR__ . '/../../vendor/autoload.php';
+$_plugin_root = dirname( dirname( __DIR__ ) );
 
 /**
- * Load WorDBless.
+ * Locate WordPress or wordpress-develop. We look in several places.
  */
-\WorDBless\Load::load();
+if ( false !== getenv( 'WP_DEVELOP_DIR' ) ) {
+	// Jetpack Monorepo environment variable.
+	$_tests_dir = getenv( 'WP_DEVELOP_DIR' );
+	if ( file_exists( "$_tests_dir/tests/phpunit/" ) ) {
+		$_tests_dir .= '/tests/phpunit/';
+	}
+} elseif ( false !== getenv( 'WP_TESTS_DIR' ) ) {
+	// WordPress core environment variable.
+	$_tests_dir = getenv( 'WP_TESTS_DIR' );
+} elseif ( file_exists( dirname( dirname( $_plugin_root ) ) . '/tests/phpunit/includes/bootstrap.php' ) ) {
+	// Installed inside wordpress-develop.
+	$_tests_dir = dirname( dirname( $_plugin_root ) ) . '/tests/phpunit/includes/bootstrap.php';
```

Comment:
This bit is why your tests are failing to find things in CI. The corresponding bit in Jetapack's boostrap is
https://github.com/Automattic/jetpack/blob/89f43a22a4624fc6e3d56c6ccf0d061b7f3893a3/projects/plugins/jetpack/tests/php/bootstrap.php#L30-L32
I _think_ with what you have here you probably need two more levels of `dirname()`.

Reply:
Your hunch was correct! It was that specific path that wasn't correct 🎉 

I'll create a separate PR to fix the path in [the example bootstrap file](https://github.com/Automattic/jetpack/blob/trunk/docs/examples/bootstrap.php) when I'm back from AFK in a couple of weeks